src/api/spec/models/obs_factory/openqa_api_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe ObsFactory::OpenqaApi do
+  let(:openqa_api) { ObsFactory::OpenqaApi.new('https://some_url.com/') }
+
+  describe '::new' do
+    it { expect(openqa_api.base_url).to eq('https://some_url.com/api/v1/') }
+  end
+
+  describe '#get' do
+
+		context 'when at least one parameter key is empty' do
```

Comment:
are these tabs? Let there be a rubocop for that :)

Reply:
fixed :smile: